cmd/root.go

```diff
@@ -41,8 +41,6 @@ func Execute() {
 }
 
 func init() {
-	cobra.OnInitialize(initConfig)
```

Comment:
So this is allowing `help` and other commands to work if the `.sops.yaml` file isn't there yet. Nice!